airflow/providers/openlineage/plugins/openlineage.py

```diff
@@ -46,7 +47,5 @@ class OpenLineageProviderPlugin(AirflowPlugin):
 
     name = "OpenLineageProviderPlugin"
     if not _is_disabled():
-        from airflow.providers.openlineage.plugins.listener import OpenLineageListener
-
         macros = [lineage_run_id, lineage_parent_id]
-        listeners = [OpenLineageListener()]
+        listeners = [get_openlineage_listener()]
```

Comment:
Why do we need this change?

Reply:
Already answered to related question. The change is to be able to get same instance of OpenLineageListener.